.github/workflows/reusable-deployment.yml

```diff
@@ -98,4 +111,4 @@ jobs:
           MSG_MINIMAL: true
           SLACK_FOOTER: ""
           SLACK_ICON: https://avatars.githubusercontent.com/u/1005263?s=200&v=4
-          SLACK_MESSAGE: "This automated script is happy to announce that a new docker image has been built for *master* with tags: [${{ steps.docker_build.outputs.tags }}] :docker:"
\ No newline at end of file
+          SLACK_MESSAGE: "This automated script is happy to announce that a new docker image has been built for *master* with tags: [${{ steps.docker_build.outputs.tags }}] :docker:"
```

Comment:
Looks like this is hard coded to "*master* we can use the environment name passed in e.g. "trunk" or "nightly"

Reply:
Now we are using `inputs.environment` instead of hard coded master.